packages/flutter_tools/test/artifacts_test.dart

```diff
@@ -18,12 +18,12 @@ void main() {
     CachedArtifacts artifacts;
 
     setUp(() {
-      tempDir = fs.systemTempDirectory.createTempSync('flutter_temp');
+      tempDir = fs.systemTempDirectory.createTempSync('flutter_tools_test.');
```

Comment:
We should try to make the names unique per test so that if we see any lingering directories, we'll know which test is the culprit. So here: `flutter_tools_artifacts_test.`